suite-native/receive/src/hooks/useAccountReceiveAddress.tsx

```diff
@@ -129,6 +134,11 @@ export const useAccountReceiveAddress = (accountKey: AccountKey) => {
                 });
                 setIsReceiveApproved(true);
             }
+        } else if (isDeviceRemembered && isViewOnlyFeatureEnabled) {
```

Comment:
But only if the device is disconnected!

Reply:
Good catch!
[ac051f6](https://github.com/trezor/trezor-suite/pull/11928/commits/ac051f606290b030f18c5369874a3756ba5815a7)